### PR TITLE
 Package Superset reducer module into a js-tool package

### LINKED ISSUE #83: Superset reducer module
It might be a good idea to package this Superset reducer module https://github.com/onaio/reveal-frontend/blob/master/src/store/ducks/superset.ts into a js-tools package, for ease of re-usability.

### PR DESCRIPTION
**Pull request by FrankApiyo**
Fixes #83

### CODE REVIEW

File: packages/superset-reducer/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    ]
+  },
+  "peerDependencies": {
+    "@types/seamless-immutable": "^7.1.8",
+    "redux": "^4.0.1",
+    "redux-thunk": "^2.3.0",
+    "seamless-immutable": "^7.1.4"
+  },
+  "dependencies": {
+    "@types/react-table": "^6.7.22",
```
Comment (moshthepitt) [inline packages/superset-reducer/package.json]: is this really a dependency?

File: packages/superset-reducer/package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "prop-types": "^15.6.1"
```
Comment (moshthepitt) [inline packages/superset-reducer/package.json]: is this really a dependency?

File: packages/superset-reducer/README.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+## Usage
+
+Using this reducer is quite simple and can be done in one of two ways:
+
+1. Use [combineReducers](https://redux.js.org/api/combinereducers) to ensure that the session reducer is loaded into your Redux store
+
+OR
+
+1. Register the session reducer so that it is added to your Redux store dynamically. You would do this in the case that you are using the [Reducer Registry](https://github.com/onaio/js-tools/tree/master/packages/reducer-registry).
```
Comment (moshthepitt) [inline packages/superset-reducer/README.md]: Did you mention session reducer by mistake?

Comment (FrankApiyo) [reply packages/superset-reducer/README.md]: yes

File: packages/superset-reducer/README.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+1. Use [combineReducers](https://redux.js.org/api/combinereducers) to ensure that the superset reducer is loaded into your Redux store
+
+OR
+
+1. Register the superset reducer so that it is added to your Redux store dynamically. You would do this in the case that you are using the [Reducer Registry](https://github.com/onaio/js-tools/tree/master/packages/reducer-registry).
+
+### sample code to register the reducer
+
+```typescript
+import supersetReducer, { reducerName as supersetReducerName } from '@onaio/session-reducer';
```
Comment (moshthepitt) [inline packages/superset-reducer/README.md]: Please replace session reducer with the correct reducer

**Pull request merged**